packages/inputs/src/Menu.js

```diff
@@ -2,12 +2,21 @@ import { MosaicClient, Param, isParam, isSelection, clausePoint } from '@uwdata/
 import { Query } from '@uwdata/mosaic-sql';
 import { input } from './input.js';
 
+/**
+ * @typedef {import('@uwdata/mosaic-core').Activatable} Activatable
+ */
+
 const isObject = v => {
   return v && typeof v === 'object' && !Array.isArray(v);
 };
 
 export const menu = options => input(Menu, options);
 
+/**
+ * A HTML select based dropdown menu input.
+ * 
+ * @implements {Activatable}
```

Comment:
Just adding this doesn't work?

```suggestion
 * @implements {import('@uwdata/mosaic-core').Activatable}
```

Reply:
Unfortunately no, it causes errors in the tests (specifically during running `tsc` in the spec package). See here for logs: https://github.com/uwdata/mosaic/actions/runs/13249311572/job/36983225553#step:7:186